specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/Trigger.json

```diff
@@ -576,17 +618,18 @@
               "format": "date-time",
               "description": "The end time for the time period for which restatement is initiated. Only UTC time is currently supported."
             },
-            "maxConcurrency": {
+            "rerunConcurrency": {
```

Comment:
I'm curious about this rename. Why did you drop `maxConcurrency`? IMO the previous name represents better the semantics of this property. Also having rerun in this properties name feels redundant with the parent name

Reply:
@drosales57 might be able to provide some explanations. But can we keep this change anyway for consistency reason? Would it be a good idea to use "x-ms-client-name" to rename it to "maxConcurrency" at SDK side?